

PACKAGE = "scout_odom"
NODE = "rviz_prod"


from dynamic_reconfigure.parameter_generator_catkin import *
import math

DEFAULT_INIT_DONE = False

gen = ParameterGenerator()

gen.add("init_done", bool_t, 0,"Required to initialize odom starting pose", default=DEFAULT_INIT_DONE)

exit(gen.generate(PACKAGE, NODE, "rvizParams"))

